zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -0,0 +1,131 @@
+		/obj/item/clothing/shoes/magboots,
+		/obj/item/blueprints,
+		/obj/item/clothing/head/space,
+		/obj/item/weapon/storage/internal
+	)
+	var/obj/item/device/radio/intercom/announce
+	var/on_store_message = "has clocked out of their shift and returned to the dormitories."
+	var/on_store_name = "Elevator Announcer"
+
+/obj/machinery/button/remote/elevator_panel/New()
+	announce = new /obj/item/device/radio/intercom(src)
+	update_icon()
+	..()
+
+/obj/machinery/button/remote/elevator_panel/trigger()
+	var/elevator_prompt
+	var/turf/T = get_turf(src)
+
+	if(world.time - cooldown_timer > cooldown)
+		cooldown_timer = world.time
+		elevator_prompt = alert(src, "Warning! Using the elevator will end your shift and remove you from the round. Are you -sure- you want to head to your dorm for the day?\n(Everyone inside the elevator with you will end their shift and be removed as well!)", "End your shift and head to the dorms?", "End shift", "Stay on shift")
+		if(elevator_prompt == "End shift")
+			for(var/obj/machinery/door/airlock/D in SSmachines.machinery)
+				if (D.id_tag == id)
+					D.close()
+					sleep(11)
+					D.lock()
+			sleep(10)	//Wait 1 seconds before proceeding
+			playsound(src, 'zzzz_modular_occulus/sound/effects/elevator_going_up.ogg', 70, 0)
+			sleep(10)
+			visible_message("<span class='notice'>The elevator hums and creaks as it makes its way to the dormitory floor.</span>")
+			sleep(185)	//Wait 6 seconds before proceeding
+			for (var/mob/living/mobsgoingup in get_area(src))
+				despawn_passenger(mobsgoingup)
+		else
+			T.audible_message("<font color = #F63812><B>Elevator Coms</B>: \"Error. The elevator is currently out of service. Please contact your local engineering department.\"</font>")
+	else
+		to_chat(usr, SPAN_DANGER("The elevator is currently in use. Please wait before you can use it again."))
+
+/obj/machinery/button/remote/elevator_call_button
+	name = "Elevator Call Button"
+	desc = "An button used to call the elevator down to the current floor."
+	var/cooldown = 24 SECONDS
+	var/cooldown_timer
+
+/obj/machinery/button/remote/elevator_call_button/trigger()
+	for(var/obj/machinery/door/airlock/D in SSmachines.machinery)
+		if (D.id_tag == id)
+			if(D.locked)
+				if(world.time - cooldown_timer > cooldown)
+					cooldown_timer = world.time
+					visible_message("<span class='notice'>The elevator has been called. Please wait...</span>")
+					playsound(src, 'zzzz_modular_occulus/sound/effects/elevator_going_down.ogg', 70, 0)
+					sleep(220)
+					for(D in SSmachines.machinery)
+						if (D.id_tag == id)
+							D.unlock()
+							sleep(10)
+							D.open()
+				else
+					to_chat(usr, SPAN_DANGER("The elevator is currently in transit. Please wait for it to reach the dormitories before calling it."))
+			else
+				D.open()
+
+
+//// This is the code for the elevator area.
+
+/area/eris/elevator
+	name = "elevator"
+	forced_ambience = list('zzzz_modular_occulus/sound/ambience/elevatormusic.ogg')
+	requires_power = 0
```

Comment:
FALSE

Reply:
(Though if this isn't the standard practice for on/off variable, feel free to ignore.)

Reply:
Most of the other stuff I saw regarding areas show `requires_power = 0`

I would not be surprised if this was just Eris or SS13 code in general being bad practice, cus I have also seen many instances of `TRUE` `FALSE` being used as well. Mixed code like that is bad etiquette, but at this point I'm unsure which one is supposed to be used.  Regardless, it works so... /shrug

Reply:
TRUE/FALSE should always be used for booleans no matter what. Eriscode is just in a sad state.

Reply:
Alright, I'll change it to TRUE/FALSE